pages/_app.js

```diff
@@ -1,7 +1,10 @@
+import Layout from '../components/Layout'
 import '../styles/globals.css'
 
 function MyApp({ Component, pageProps }) {
-  return <Component {...pageProps} />
+  return <Layout>
```

Comment:
As you have more then one line here, you can add parenthesis here 
```suggestion
  return (
  <Layout>
  ...
  </Layout>
  )
```